setup.py

```diff
@@ -34,7 +34,7 @@
     author_email='danjac354@gmail.com',
     description='Sendmail package for Pyramid',
     long_description='\n\n'.join([README, CHANGES]),
```

Comment:
The missing `Changelog` bit above hurts the long description here.

Reply:
This should be fixed in 18257b2